pkg/reconciler/clusterbuilder/clusterbuilder_test.go

```diff
@@ -335,9 +354,11 @@ func testClusterBuilderReconciler(t *testing.T, when spec.G, it spec.S) {
 				kreconciler.KeyForObject(clusterStore), expectedBuilder.NamespacedName()))
 			require.True(t, fakeTracker.IsTracking(
 				kreconciler.KeyForObject(clusterStack), builder.NamespacedName()))
-
 			require.True(t, fakeTracker.IsTrackingKind(
 				kreconciler.KeyForObject(clusterBuildpack).GroupKind, builder.NamespacedName()))
+			// TODO: fix test
+			//require.True(t, fakeTracker.IsTrackingKind(
+			//	kreconciler.KeyForObject(clusterExtension).GroupKind, builder.NamespacedName()))
```

Comment:
I have no idea what this is testing or why it's failing. Maintainers, please help!

Reply:
Same as https://github.com/buildpacks-community/kpack/pull/1332#discussion_r1376769422, with the only difference being that it only watches for ClusterExtensions instead of both Extension and ClusterExtension